components/Avtar.css

```diff
@@ -0,0 +1,209 @@
+@import url('https://fonts.googleapis.com/css2?family=Merriweather:ital,wght@1,900&family=Oooh+Baby&family=Open+Sans:wght@300&display=swap');
+/* css for navigation */
+
+:root{
+    --primary-color:#1E40AF;
+    --secondary-color:#BFDBFE;
+    --third-color:black;
+}
+body{
+    overflow-x: hidden;
+    overflow-y: hidden;
+    margin: 0;
+}
+#navigation{
+    background-color: var(--primary-color);
+    height: 5rem;
+    width: 100%;
+}
+#navlabel{
+    font-size: 1.9rem;
+    color: white;
+    position: relative;
+    left: 2rem;
+    top: 1.5rem;
+}
+#navItems{
+    display: flex;
+    justify-content: end;
+    position: relative;
+    right: 30rem;
+    top: -1rem;
+}
+.navItem{
+    text-decoration: none;
+    padding: 2rem 1rem;
+    margin-top: -2rem;
+    color: white;
+    font-size: 1.5rem;
+}
+.fa-github{
+    position: relative;
+    height: 1.2rem;
+    width: 1rem;
+    top:-1.6rem;
+    left: 2.3rem;
+    color: white;
+}
+.github{
+    color: white;
+    text-decoration: none;
+    margin-top: 1.9 rem;
+    font-size: 19px;
+}
+#githubid{
+    position:relative;
+    top:1rem;
+    left: 25rem;
+}
+/* outer div  */
+#outerDiv{
+    height: 34.5rem;
+    width: 100%;
+    
+}
+/* sidebar */
+.sidebarItem{
+    list-style: none;
+    text-decoration: none;
+    padding: 1rem;
+    margin-left: -1rem;
+}
+.sidebarItem:hover{
+    background-color: gainsboro;
+    border-radius: 0.5rem;
+    color: var(--primary-color)
+}
+#sidehead{
+    display: flex;
+    justify-content: center;
+    font-size: 1.5rem;
+}
+#sidebar{
+    height: 34.5rem;
+    width: 15rem;
+    overflow-y: scroll;
+    overflow-x: hidden;
+    position: relative;
+    top: -34.6rem;
+    box-shadow: 1px 2px 1px 0.1px gray;
+}
+#sidebar::-webkit-scrollbar{
+    width: 0.4rem;
+    background:var(--primary-color);
+
+}
+#sidebar::-webkit-scrollbar-thumb{
+    border-radius: 0.5rem;
+    height: 30rem;
+    background-color:gray;
+    
+}
+.AnkerItem{
+    text-decoration: none;
+    font-size: 1.2rem;
+    color: black;
+
+}
+.list{
+    margin-top: -1rem;
+}
+/* #Avtar */
+#img1{
+    border-radius: 50%;
+    width: 200px;
+    height: 12rem;
+    position: relative;
+    left: 3rem;
+    top: 1rem;
+}
+#img2{
+    border-radius: 50%;
+    width: 150px;
+    height: 8rem;
+    position: relative;
+    left: 3rem;
+    top: 1rem;
+
+}
+#img3{
+    border-radius: 50%;
+    width: 100px;
+    height: 5rem;
+    position: relative;
+    left: 3rem;
+    top: 1rem;
+}
+#img4{
```

Comment:
Try to use self explanatory names for for classes or Ids

Reply:
You can use `avatar-lg, avatar-md, avatar-sm` for large , medium and small avatar images.